### PR TITLE
Feature/react as root panorama new api

### PR DESCRIPTION
**Pull request by royderks**
Also contains the merge with the develop branch, TODO: refactor more to delete Angular

### CODE REVIEW

File: modules/straatbeeld/straatbeeld-config.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 export default {
-    STRAATBEELD_ENDPOINT_ALL: 'panorama/recente_opnames/alle/',
-    STRAATBEELD_ENDPOINT_YEAR: 'panorama/recente_opnames/',
+    STRAATBEELD_ENDPOINT_PREFIX: 'panorama/panoramas',
```
Comment (aditudorache) [inline modules/straatbeeld/straatbeeld-config.js]: merge this definition with the one in src/shared/services/panorama-api/panorama-api.js ?

File: modules/detail/components/detail/templates/dcatd/_datasets.scss [hunk 2]
```diff
@@ -17,14 +17,16 @@
     }
 
     &__right {
+      display: flex;
```
Comment (aditudorache) [inline modules/detail/components/detail/templates/dcatd/_datasets.scss]: Does this work in IE11?

Comment (royderks) [reply modules/detail/components/detail/templates/dcatd/_datasets.scss]: Yes https://caniuse.com/#feat=flexbox

File: modules/straatbeeld/straatbeeld-config.js (hunk 1 above)
Comment (royderks) [reply modules/straatbeeld/straatbeeld-config.js]: Fair point! This piece is no longer used and can be deleted

File: src/panorama/containers/PanoramaContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+const mapStateToProps = (state) => ({
+  panoramaState: getPanorama(state),
+  hotspots: getHotspots(state),
+  isPrint: isPrintMode(state),
+  // Todo: DP-6312: get the location from selection reducer
+  locationString: getMapCenter(state).join(',')
+});
+
+const mapDispatchToProps = (dispatch) => bindActionCreators({
+  doClose: toDataSearchLocation,
```
Comment (aditudorache) [inline src/panorama/containers/PanoramaContainer.jsx]: doClose should return to the previous view before opening the panorama, panorama is just a view mode on top of another state/route.

**Pull request closed without merging**